tests/libcxx/expected_results.txt

```diff
@@ -1381,3 +1381,9 @@ std/containers/sequences/vector/vector.modifiers/assert.push_back.invalidation.p
 # This test is marked as `REQUIRES: has-unix-headers, libcpp-has-abi-bounded-iterators-in-std-array`.
 # Listing these on separate lines would allow magic_comments.txt to recognize it.
 std/containers/sequences/array/assert.iterators.pass.cpp:9 SKIPPED
+
+# These tests emit C5321, which warns when the resolution to core issue 1656 affects a u8 string literal.

```

Comment:
Do we want to say "CWG-1656" instead of "core issue 1656" here? Or... as this PR is merely mirroring so any wanted style change should be in a later PR?

Reply:
Yes, that would be a good style change, and yes, it should be a followup. Thanks!

Reply:
The toolset update has to patch expected_results.txt extensively so I'm fixing it there.